servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HttpClients.java

```diff
@@ -221,7 +221,7 @@ private HttpClients() {
     public static SingleAddressHttpClientBuilder<HostAndPort, InetSocketAddress> forResolvedAddressViaProxy(
             final HostAndPort address, final HostAndPort proxyAddress) {
         return DefaultSingleAddressHttpClientBuilder.forResolvedAddressViaProxy(address,
-                new InetSocketAddress(address.hostName(), address.port()), proxyAddress);
+                toInetSocketAddress(address), proxyAddress);
```

Comment:
@Scottmitch `createUnresolved` doesn't work when you pass an IP-address. I had to create a utility for this conversion.